components/Profile/ProfileStatsBox.tsx

```diff
@@ -87,12 +82,14 @@ const ProfileStatsBox = ({
 
       {/* Level and progress bar */}
       <View className="mb-2 flex flex-row justify-between">
-        <Text className="font-sans-bold text-primary_custom">
+        <Text
+          className="caption-lg-semibold"
```

Comment:
This doesn't seem to be a preset in the typography Figma. Possibly we removed this preset when we removed your changes in `theme/typography.ts`. You probably need to use `caption-sm-semibold` here.